### DISCUSSION TITLE
(test-case-filter): command line option

### DISCUSSION THREAD

**Discussion by followynne** [stryker-mutator/stryker-net#2247] [Ideas]
Hello all!

I checked the documentation about the [test-case-filter](https://stryker-mutator.io/docs/stryker-net/configuration/#test-case-filter-string) parameter and noticed that the CLI option isn't available.

I read the original [PR](https://github.com/stryker-mutator/stryker-net/pull/1669#discussion_r693339231) and the CLI option was mentioned as removed. I think this option should be settable from the CLI, as tests filter can be applied dynamically in a CI/CD env.

(Ie: in our CI/CD env we wanted to combine stryker with parallel tests runs, that we do by using the `dotnet tests --filter` option)

Is it fine if I create a small PR that adds the CLI option? Or is this parameter to be considered a static one? 

Thanks! 😄

**Comment by rouke-broersma** [comment]
Can you elaborate more about what's actuality dynamic about your test case filter? Is the used filter actually being calculated in the pipeline?

**Reply by followynne** [reply]
Exactly, I forgot to explain sorry! 
There is a previous step that calculates the tests slices, based on how many parallel agents will be used, and provides this info to apply the filters.

**Reply by rouke-broersma** [reply]
I'm having a hard time seeing how that would work for mutation testing. Mutation testing is based around mutations, and the applicable unit tests are ran against the mutants. If you're slicing your unit tests you're guaranteed invalid results in the end.

**Reply by followynne** [reply]
I get your point, thanks for your answer. We'll probably try to parallel projects-runs, instead of test-slices-runs :)
